verity/src/helm/values.yaml

```diff
@@ -20,16 +25,48 @@ deployment:
       periodSeconds: 5
       timeoutSeconds: 2
       failureThreshold: 2
+  resources: # TODO tune the values
+    requests:
+      memory: 768Mi
+      cpu: 250m
+    limits:
+      memory: 1024Mi
+      cpu: 1000m
+  useAffinity: false # TODO requires better configuraton
+
```

Comment:
I think we should delete hardcoded values here at all. The values for requests/resources are assigned in Ansible as Helm-values.

Reply:
removed